pkg/console/starter/starter.go

```diff
@@ -232,10 +232,13 @@ func RunOperator(ctx context.Context, controllerContext *controllercmd.Controlle
 	)
 
 	consoleRouteController := route.NewRouteSyncController(
+		// top level config
+		configClient.ConfigV1(),
 		// operator config
 		operatorConfigClient.OperatorV1().Consoles(),
 		routesClient.RouteV1(),
 		kubeClient.CoreV1(),
+		kubeClient.CoreV1(),
```

Comment:
I think we only need to pass `kubeClient.CoreV1()` once.

Reply:
Ah, I see what we did, we have our clients slightly misnamed.  Hmm. And we do it in all our controllers. :)